gcs-fetcher/README.md

```diff
@@ -1,7 +1,7 @@
 # GCS Fetcher
 
-** This builder is experimental and currently very likely to change in breaking
-ways at this time. **
+** This builder is experimental and is very likely to change in breaking ways at
```

Comment:
How about adding this as a warning:

Warning: This builder is experimental and is likely to change in breaking ways at this time.

Reply:
Adding additional comments here since GitHub is not letting me add comments to unchanged lines.

Line 12: s/which/that

Line 12: format of the manifest is a mapping

13: 
the destination file path...
s/gcs/Cloud Storage
16: The following code shows an example source manifest:
32: 
To process the above manifest, the GCS Fetcher tool, ...
"the tool each element" - not sure what you're trying to say here :)